apps/server/src/modules/board/uc/column.uc.ts

```diff
@@ -42,8 +42,7 @@ export class ColumnUc extends BaseUc {
 		this.logger.debug({ action: 'createCard', userId, columnId });
 
 		const column = await this.columnService.findById(columnId);
-		// TODO students should not be allowed to create cards
-		await this.checkPermission(userId, column, Action.read);
+		await this.checkPermission(userId, column, Action.write);
```

Comment:
this was technically not part of the ticket, but it's a small bug and should be fixes asap anyway